WordPress/Docs/PHP/DontExtractStandard.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Don't exctract"
+    >
+    <standard>
+    <![CDATA[
+    Restricts the usage of `extract()` php function. It makes code harder to debug and harder to understand and may cause conflicts on variables names.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Similarly named functions or methods are fine">
+        <![CDATA[
+$var_array = array(
+    "title"    => "My title",
+    "content"  => "My content",
+    "ID"       => 123
+);
+my_extract( $var_array, EXTR_OVERWRITE );
```

Comment:
Rather than giving a valid example as being "Use a similarly name function", I'd suggest demonstrating how to use keys from the `$var_array` directly.  I'd also suggest renaming the variable to something a little more contextual.

```php
$post_data = array(
    'title'   => 'My title',
    'content' => 'My content',
    'ID'      => 123
);
<em>echo $post_data['title'];</em>
```

The `<em>` `</em>` is used here as that's the bit we're emphasising. It should be added around the `extract()` call in the Invalid section too.

Reply:
I agree with @GaryJones' remark here. The "Valid" example should show the correct way of doing it, not what's "tolerated" to prevent false positives.

When updating the code sample, please also update the "Valid" description.